Formula/d/dependabot-cli.rb

```diff
@@ -0,0 +1,31 @@
+class DependabotCli < Formula
+  desc "Tool for testing and debugging Dependabot update jobs"
+  homepage "https://github.com/dependabot/cli"
+  url "https://github.com/dependabot/cli/archive/refs/tags/v1.54.0.tar.gz"
+  sha256 "ae496c65e4f947cca1251c9f786fbcc1e5e1a079af7fee08b893265e69d39295"
+  license "MIT"
+
+  head "https://github.com/dependabot/cli.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -X github.com/dependabot/cli/cmd/dependabot/internal/cmd.version=#{version}
+    ]
+    cd "cmd/dependabot" do
+      system "go", "build", *std_go_args(output: bin/"dependabot", ldflags:)
+    end
+
+    generate_completions_from_executable(bin/"dependabot", "completion")
+  end
+
+  test do
+    assert_match "dependabot version #{version}", shell_output("#{bin}/dependabot --version")
```

Comment:
Is there a more functional test than a version test that can be used instead?

Reply:
See https://github.com/Homebrew/homebrew-core/pull/181253#discussion_r1718317421

Reply:
Sure, we can check that it bails when Docker isn't available, since it requires `DOCKER_HOST` to point at a running Docker Engine. Will address in the linked comment.